runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -418,10 +412,6 @@ task validatesRunnerV2Streaming {
     name: 'validatesRunnerV2TestStreaming',
     pipelineOptions: runnerV2PipelineOptions + ['--streaming=true'],
     excludedCategories: [
-      // Not supported in Dataflow streaming mode.
-      'org.apache.beam.sdk.testing.UsesRequiresTimeSortedInput',
```

Comment:
This I am confused about. I did not think that Dataflow supported requiring time-sorted input in streaming either runner v1 or v2.

Reply:
We moved the `UsesRequiresTimeSortedInput ` to the common exclude categories, which is for both streaming and batch.